src/components/cqlLibraryList/CqlLibraryList.tsx

```diff
@@ -355,10 +355,23 @@ export default function CqlLibraryList({ cqlLibraryList, onListUpdate }) {
                 <button
                   data-testid={`create-new-draft-${selectedCQLLibrary.id}-button`}
                   onClick={() => {
-                    setCreateDraftDialog({
-                      open: true,
-                      cqlLibrary: selectedCQLLibrary,
-                    });
+                    cqlLibraryServiceApi
```

Comment:
can we not write the logic code for onclick in separate function as we are calling the api and catching the errors

Reply:
just pushed a change to move the logic to its own function.
Also, this will (hopefully soon) become irrelevant as we should refactor how this works for this tech debt item: [MAT-7185](https://jira.cms.gov/browse/MAT-7185)